card/db/migrate_core_cards/20150317162412_bootstrap_themes.rb

```diff
@@ -2,12 +2,33 @@
 
 class BootstrapThemes < Card::CoreMigration
   def up
-    Card.create! :name=>'raw bootstrap skin', :type_code=>:skin, :content=> "[[style: bootstrap]]\n[[style: jquery-ui-smoothness]]\n[[style: functional]]\n[[style: standard]]\n[[style: right sidebar]]\n[[style: bootstrap cards]]"
-    %w{amelia simpex cerulean lumen darkly readable holo superhero yeti cosmo cyborg spacelab google_plus facebook}.each do |theme|
+    Card.create! :name=>'themeless bootstrap skin', :type_code=>:skin, :content=> "[[style: bootstrap]]\n[[style: jquery-ui-smoothness]]\n[[style: cards]]\n[[style: right sidebar]]\n[[style: bootstrap cards]]"
+    %w{cerulean cosmo cyborg darkly flatly journal lumen paper readable sandstone simplex slate spacelab superhero united yeti }.each do |theme|
       Card.create! :name=>"theme: #{theme}", :type_code=>:css, :codename=>"theme_#{theme}"
-      Card.create! :name=>"#{theme} skin", :type_code=>:skin, :content=>"[[raw bootstrap skin]]\n[[theme: #{theme}]]"
+      Card.create! :name=>"#{theme} skin", :type_code=>:skin, :codename=>"#{theme}_skin", :content=>"[[themeless bootstrap skin]]\n[[theme: #{theme}]]"
     end
```

Comment:
renamed raw to themeless, because it seemed to do a job of communicating the relationships.